docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,52 @@
+Read the Docs for Business subscription management
+==================================================
+
+We want to make it easy to manage your billing information.
+You can manage or cancel subscriptions from the :guilabel:`Settings > Subscriptions` page of the :term:`dashboard`.
+
+Managing your subscription
+--------------------------
+
+* Go to https://readthedocs.com/organizations/.
+* Select your organization from this list.
+* Select :guilabel:`Settings` at the top of the page.
+* Select the :guilabel:`Subscription` tab on the left.
+* Click :guilabel:`Manage Subscription`.
+
+This action will take you to the Stripe billing portal where you can manage your subscription.
+
+.. note::
+    You will need to be an organization owner to view subscription information.
+    If you do not have permission,
+    you can ask one of your existing organization owners to make any required changes.
+
+Cancelling your subscription
+----------------------------
+
+Cancelling your subscription can be done in the same way as the above instructions.
+You will be billed for the remainder of the current billing period,
+and then not charged again.
+
+**We can not cancel subscriptions just by sending us an email,
+as email is an insecure method of verifying a user's identity.**
+If you email us about this,
+we require you to verify your identity by logging into your Read the Docs account and submitting an official support request there.
```

Comment:
I'm also fine saying 'An organization owner must make this change'. The majority of the time I get these requests, the user doesn't even seem to have an RTD account.

Reply:
Yea, we mention that at the top as well.